src/config.rs

```diff
@@ -212,17 +202,30 @@ fn merge_millis_setting_if_set(
     )
 }
 
+/// Get the peers as a Vec<PeerId> from settings
+pub fn get_peers_from_settings<S: std::hash::BuildHasher>(
```

Comment:
Is there a reason we're specifying the particular hasher that we'll accept here? If so, can we add a comment to that effect? Otherwise, it should be possible to just accept `HashMap<String, String>`.

Reply:
Clippy told me to generalize it, because there's nothing in the code that's specific to HashMap. I can ignore the lint or add a comment if you think that's necessary though.